src/screens/AddSiteFormScreen.js

```diff
@@ -0,0 +1,445 @@
+import React, {Component} from 'react';
```

Comment:
Most of this screen is the same as what was previously in the modal in `AddSiteScreen` with the exception of the checkbox logic as well as removing the "close" modal button, and moving the "Reset" button to the top right nav instead of in the form itself.